lib/Runtime/Library/arm/arm_CallFunction.asm

```diff
@@ -70,48 +68,51 @@
     ;  - output: r4 = total number of BYTES probed/allocated.
     blx     __chkstk            ;now r4 = the space to allocate, r5 = actual space needed for values on stack, r4 >= r5.
 
+#if defined(_CONTROL_FLOW_GUARD)
+    ldr r6,[sp, #32] ; r6 = ep
+#else
```

Comment:
Took me quite a while to understand what "ep" meant. Please make this more clear. Also fix spacing to match existing code.